### PR TITLE
Attempt programming questions

### PR DESCRIPTION
**Pull request by lowjoel**

### CODE REVIEW

File: app/helpers/application_cocoon_helper.rb [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
-  #   @option html_options [Symbol] :find_using The jQuery traversal method to allow node selection
-  #     relative to the link. Example values: `closest`, `next`, `children`, etc. Default: absolute
-  #     selection.
-  #   @option html_options [Symbol] :selector The jQuery selector to be used with +:find_using+ to
-  #     determine where to insert the new node. Default: parent node.
-  #   @option html_options [Symbol] :find_selector The alias of +:selector+, you should use this
+  #   @option html_options [String] :find_using The jQuery traversal method to allow node selection
+  #     relative to the link. Example values: +this+, +closest+, +next+, +children+, etc.
+  #     Default: absolute selection.
+  #   @option html_options [String] :selector The jQuery selector to be used with +:find_using+ to
```
Comment (allenwq) [inline app/helpers/application_cocoon_helper.rb]: Why all these were changed to String?

Comment (lowjoel) [inline app/helpers/application_cocoon_helper.rb]: Because the value of each of these options are strings, not symbols

Comment (allenwq) [inline app/helpers/application_cocoon_helper.rb]: But we will still passing symbols to it ?

Comment (lowjoel) [inline app/helpers/application_cocoon_helper.rb]: what do you mean?

File: app/views/course/assessment/answer/programming/_programming.html.slim [hunk 2]
```diff
@@ -0,0 +1,10 @@
+- question = programming.question.specific
+- file_fields_path = 'course/assessment/answer/programming/file_fields'.freeze
+h2 = format_inline_text(question.title)
+= format_html(question.description)
+
+div.files
+  = link_to_add_association t('.add_file'), f, :files, partial: file_fields_path,
+                            find_selector: :this, insert_using: :after
```
Comment (allenwq) [inline app/views/course/assessment/answer/programming/_programming.html.slim]: We will still using symbols as parameter?

File: app/helpers/application_cocoon_helper.rb (hunk 1 above)
Comment (lowjoel) [inline app/helpers/application_cocoon_helper.rb]: these are JavaScript values, not Ruby values, so actually symbol or string doesn't make a difference. Grepping shows strings are used more than symbols

Comment (allenwq) [inline app/helpers/application_cocoon_helper.rb]: Yah, even in ruby most of time both are supported.

Anyway it's a small issue.

File: app/views/course/assessment/answer/programming/_programming.html.slim (hunk 2 above)
Comment (lowjoel) [inline app/views/course/assessment/answer/programming/_programming.html.slim]: woops, I changed it back to test the output

**Pull request merged**